src/shared/components/instant-download/instant-download-gene/InstantDownloadGene.scss

```diff
@@ -15,37 +15,16 @@
   margin-left: 15px;
 }
 
-.checkboxUnchecked {
-  background-color: transparent;
-  border-color: $grey;
+.checkboxDefault {
+  height: 14px;
+  width: 14px;
 }
```

Comment:
- is this something that can be controlled through a CSS variable on the parent CSS class?
- does Andrea really want 14 pixels now? doesn't it contradict her style guide? please check with her

Reply:
Yes we should be using the CSS variables. As for the different size, I'll check with her and see. We probably can remove this class like you suggested for the `CustomDownloads` checkbox.

Reply:
✅